hack/gcr-push.sh

```diff
@@ -47,24 +47,34 @@ function highest_release() {
     done
 }
 
-if [ "$BRANCH" == "master" ]; then
+if [[ "$BRANCH" == "master" ]]; then
```

Comment:
Did you mean to have double bracket here?

Reply:
Yep. I should update the other branch below this, too.